COMPATIBILITY.md

```diff
@@ -77,7 +81,7 @@ Entries list the class name serving as the entry point to the relevant functiona
 - [X] `bitbucket`: supported as of 1.1.2
 
 ## Clouds
-
+The following Clouds are delivered:
```

Comment:
The sentence is vague a bit. "Workflow supports the on-demand nodes provisioning from the following Cloud providers" is better IMHO